blog/docs/events/virtual-office-hour-12-2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09"
+type: "blog"
+date: 2022-12-01
+linkTitle: "First Knative Virtual Office Hour / Contrib Fest 2022/12/09"
+description: "Join the Zoom Knative First Knative Virtual Office Hour / Contrib Fest on 2022/12/09"
+---
```

Comment:
```suggestion
**Authors: [Alek Slominski](https://aslom.net/), [Evan Anderson](https://off-by-one.dev/), and Paul Schweigert**

**Date: 2022-12-01**

# First 🌱 Knative Virtual Office Hour / Contrib Fest 2022/12/09
```

this is the new format we're using for blog headers that we switched to a little bit ago

Reply:
Updated to the new blog format.

Let make separate PR to docs/blog/config/nav.yml and docs/blog/docs/index.md when this PR is merged?

Reply:
I'd do at least the nav.yaml one in this PR, otherwise the blog post won't be visible.